lib/appscale.py

```diff
@@ -131,6 +134,24 @@ def get_locations_json_file(self, keyname):
     return json_file
 
 
+  def get_nodes(self, keyname):
+    """Retrieve a list of the running nodes.
```

Comment:
Put a whitespace after """ and before the text.
Do the same in the rest of the docstrings.